contents/handbook/engineering/feature-pricing.md

```diff
@@ -30,14 +30,20 @@ Sometimes these principles still leave room for questions – what, if anything,
 
 For these types of questions, we've defined a runbook for deciding which plans, and at what limits, features should be assigned to.
 
-### We should match the cheapest competitor
+### We should slightly undercut the cheapest competitor
 
-In general, we should roughly match the pricing of the cheapest big competitor for that product, so long as the unit economics make sense, to make it a no-brainer to use PostHog.
+In general, we should slightly undercut the pricing of the cheapest big competitor for that product, so long as the unit economics make sense, to make it a no-brainer to use PostHog. To qualify for this, a competitor must be _making actual revenue_ at significant scale - we won't match the pricing random startups or new products at existing competitors offer, since these products and GTMs aren't mature yet.
 
 We can do this because we can upsell customers multiple of our other products. The total ACV is higher even if the per-product ACV is lower. 
 
 It's better for customers because they get all these tools that are well integrated for the cheapest possible price.
 
+### Every product should be priced separately
+
+Whenever we build a product, like feature flags, or product experimentation, we should have a specific price for that product by itself. Being consistent here is less confusing than randomly combining products for example, even thouigh it will sometimes mean more items to explain to a customer.
```

Comment:
```suggestion
Whenever we build a product, like feature flags, or product experimentation, we should have a specific price for that product by itself. Being consistent here is less confusing than randomly combining products for example, even though it will sometimes mean more items to explain to a customer.
```